ios-build-support-lib.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
```

Comment:
Can this file go somewhere other than at the top-level?  I feel like it belongs in `support-lib/`.

Reply:
I have created the script at the same level as the `CMakeLists.txt` that builds the support library. Should I move it also?

Reply:
Hmm, good point, which I didn't realize since I don't use CMake.  If I were doing it from scratch, I feel like that CMakeLists.txt file belongs in the subdir too, along with the .gyp file.  I'd still suggest moving the script, though.  If we had some sort of a `bin` or `scripts` dir it would belong there, but `support-lib` seems best for now.